.user.ini

```diff
@@ -1,5 +1,5 @@
 ini_set('session.cookie_httponly', 1); // Prevents javascript XSS attacks aimed to steal the session ID
 ini_set('session.use_only_cookies', 1); // Session ID cannot be passed through URLs, so people cannont share/bookmark them
 ini_set('session.cookie_secure', 1); // Uses a secure connection (HTTPS) if possible
-ini_set('session.gc_maxlifetime', (7 * 24 * 3600)); //One week duration
```

Comment:
This file is not in ini format.... are we sure it's being run when the webserver is started?

Reply:
See the examples here: https://www.php.net/manual/en/configuration.file.php